### ISSUE TITLE
Android start/stop beep sound

### ISSUE THREAD

**Issue by Facuiguacel** [JamesBrill/react-speech-recognition#213]
Hi, has anyone been able to work around the annoying android beep sound when speech starts and stops? Even on continuous mode it beeps everytime it detects voice and beeps again when you stop talking.

**Comment by prafuldivani** [comment]
Yes, Faced same issue. @Facuiguacel Have you found any solution to this ?

**Comment by Facuiguacel** [comment]
There's no solution, the beeping sound is from Android's native Speech Recognition service and cannot be avoided. > Yes, Faced same issue. @Facuiguacel Have you found any solution to this ?
